tola_management/programadmin.py

```diff
@@ -608,7 +608,7 @@ def export_audit_log(self, request, pk=None):
         program = Program.objects.get(pk=pk)
         workbook = Workbook()
         workbook.remove(workbook.active)
-        ws = workbook.create_sheet(_('Change log'))
+        ws = workbook.create_sheet(_('Program change log'))
```

Comment:
Translator comment as above.

Reply:
@sanjuroj I added a translator comment. Can you make sure that is the correct syntax?

Reply:
Yep, looks good.  If you're unsure, you can always run the makemessages command again and look for the comment in the .po file (either with a plain text editor or the POEdit software).  I sometimes still have to do that with JS files or Django template files, because the syntax can get a bit weird.